mne/tests/test_source_estimate.py

```diff
@@ -881,4 +886,45 @@ def test_vec_stc():
     assert_array_equal(normal.data[:, 0], [1, 2, 0, np.sqrt(3)])
 
 
+@requires_sklearn
+@testing.requires_testing_data
+def test_vol_connectivity():
+    """Test volume connectivity."""
+    from scipy import sparse
+    vol = read_source_spaces(fname_vsrc)
+
+    assert_raises(ValueError, spatial_src_connectivity, vol, dist=1.)
+
+    connectivity = spatial_src_connectivity(vol)
+    n_vertices = vol[0]['inuse'].sum()
+    assert_equal(connectivity.shape, (n_vertices, n_vertices))
+    assert_true(np.all(connectivity.data == 1))
+    assert_true(isinstance(connectivity, sparse.coo_matrix))
+
+    connectivity2 = spatio_temporal_src_connectivity(vol, n_times=2)
+    assert_equal(connectivity2.shape, (2 * n_vertices, 2 * n_vertices))
+    assert_true(np.all(connectivity2.data == 1))
+
+
+@requires_sklearn
+@requires_nibabel()
```

Comment:
@larsoner remove nibabel?

Reply:
No the `vol_mask` test uses `nibabel` because it uses `as_volume`. I did remove it for the test above this one, which no longer needs it.